src/_includes/feature_lists/self-hosted.njk

```diff
@@ -1,6 +1,35 @@
 {%
     set sections = [{
-        label: "Features",
+        label: "No-Ops Node-RED",
```

Comment:
At what stage will we unify the too feature_list files? :)

Reply:
I wish I could say at this stage, but I spent some time trying to do it, and it wasn't working. I can give it another try some other time. I'd like to move on to other items on my to-do list if that's ok 🙂